### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by danieljue**
Added a link to a simple helper script for prioritizing downloads

Documentation change only so PR against master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ Implemented:
 ## User Scripts
 This is a list of useful user scripts, generously created from folks like you to extend this project and its functionality. This is your time to shine, [read this](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#user-scripts) then open a PR to add your script here.
 
-- Example 1
+[danieljue/ta_dl_page_script](https://linktoyourrepo.com): Helper browser script to prioritize a channels' videos in download queue.
```
Comment (Krafting) [inline README.md:193]:
```suggestion
[danieljue/ta_dl_page_script](https://github.com/danieljue/ta_dl_page_script): Helper browser script to prioritize a channels' videos in download queue.
```

You probably mean this instead ?

Comment (danieljue) [reply README.md:193]: Oops yes! sorry about that.

**Comment by lamusmaser** [comment]
LGTM, @bbilly1 for review.

**Comment by bbilly1** [comment]
Nice, looks useful. Great teamwork!

**Pull request merged**